polyfill/lib/plainmonthday.mjs

```diff
@@ -55,13 +55,13 @@ export class PlainMonthDay {
     }
   }
 
-  get month() {
+  get monthCode() {
     if (!ES.IsTemporalMonthDay(this)) throw new TypeError('invalid receiver');
-    const result = GetSlot(this, CALENDAR).month(this);
+    const result = GetSlot(this, CALENDAR).monthCode(this);
     if (result === undefined) {
-      throw new RangeError('calendar month result must be a positive integer');
+      throw new RangeError('calendar monthCode result must be a string');
     }
-    return ES.ToPositiveInteger(result);
+    return ES.ToString(result);
```

Comment:
Hmmm, I was assuming that we'd require the calendar to return an actual string, not something that could be converted to a string. For example, if the calendar returns `2`, that seems like a bug in the calendar to me, even though it can be converted to `'2'`. What do you think?

Reply:
Should consider for #1319 as well, not just monthCode - agreed, what do you think?

Reply:
I'll read the notes back, but I seem to remember we decided to defensively call these conversion methods on the return value from the calendar.

Reply:
Defense sounds right, I was just assuming that the defense would be throwing not casting.

Reply:
Here's what I found in the notes:
We did:
> SFC: I think for output it doesn't matter that much. We could pass whatever the calendar gives through the String or Number constructor. The point of having this would be to prevent userland calendar authors from doing something they're not supposed to do.
> JGT: That actually sounds good to me, because the callers of Temporal methods can be sure of what type they get back. That seems like a good thing.